IHP/QueryBuilder.hs

```diff
@@ -753,6 +770,33 @@ innerJoin (name, name') queryBuilderProvider = injectQueryBuilder $ JoinQueryBui
         rightJoinColumn = (Text.encodeUtf8 . fieldNameToColumnName) (symbolToText @name')
 {-# INLINE innerJoin #-}
 
+-- | Index the values from a table with values of a field from a table joined by 'innerJoin' or 'innerJoinThirdTable'. Useful to get, e.g., the comments to a set of posts in such a way that the assignment of comments to posts.
+--
+--
+-- __Example:__ Fetch a list of all comments, each paired with the id of the post it belongs to.
+-- labeledTags <-
+--   query @Tag
+--      |> innerJoin @Tagging (#id, #tagId)
+--      |> innerJoinThirdTable @Post @Tagging (#id, #postId)
+--      |> labelResults @Post #id
+--      |> fetch
+---- > -- SELECT posts.id, tags.* FROM comments INNER JOIN taggings ON tags.id = taggings.tagId INNER JOIN posts ON posts.id = taggings.postId 
```

Comment:
In this comment the haddock format is a bit broken. Take a look at e.g. https://github.com/digitallyinduced/ihp/blob/master/IHP/AuthSupport/Authorization.hs#L14

After the `__Example:__ Fetch a list of all comments, each paired with the id of the post it belongs to.` we need a new line otherwise haddock will have troubles parsing this.

Also the code samples need a `>` in front :)

Reply:
Oops, right. Should be fixed now. Also corrected the content of the comment to match the new example.